app/Models/Employee.php

```diff
@@ -29,16 +29,21 @@ public function __construct() {
 	/**
 	 * Get query to be used in views in the loop
 	 *
+	 * @param array $args Arguments.
+	 *
 	 * @return \WP_Query  query object to be used in loop
 	 */
-	public function get_query() {
-		return $this->archive_query( array(
+	public function get_query( $args ) {
+		$default = array(
 			'post_type'      => Post_Type\Employee::$ID,
 			'post_status'    => Post_Type\Employee::STATUS_PUBLISH,
 			'order'          => Post_Type\Employee::SORT_DESC,
 			'orderby'        => Post_Type\Employee::ORDERBY_DATE,
 			'posts_per_page' => 4,
-		), __METHOD__ );
-	}
+		);
 
+		$args = wp_parse_args( $args, $default );
+
+		return $this->wp_query( $args, __METHOD__ );
```

Comment:
we had "archive_query" here. if you remove this - you will break pagination on usual employees page